datareservoirio/_logging.py

```diff
@@ -28,11 +28,20 @@ def wrapper(self, *args, **kwargs):
         try:
             return func(self, *args, **kwargs)
         except Exception as e:
-            properties = {
-                "custom_dimensions": {
-                    "drio_package": f"python-datareservoirio/{drio.__version__}"
+            if os.getenv(ENV_VAR_ENGINE_ROOM_APP_ID) is not None:
+                engine_room_app_id = os.environ[ENV_VAR_ENGINE_ROOM_APP_ID]
+                properties = {
+                    "custom_dimensions": {
+                        "drio_package": f"python-datareservoirio/{drio.__version__}",
+                        "EngineRoom App ID": engine_room_app_id,
```

Comment:
![image](https://github.com/4Subsea/drio-python/assets/142307980/3625dbe3-1272-4626-99b9-be4f2ffa20cc)

Then it will be a difference in how we make the custom dimensions in the timer and the log_exceptions but that is maybe okay?

Reply:
It seems to be total mess - some are `with-dashes`, some with `fancy_underscores`, and some `upperCamelCase` :D I think it's worth doing as library does automatically, so `upperCamelCase`.

Reply:
Okay, should I change the others as well or keep them?

Reply:
I'd not change existing ones, but let's try to keep creating new dimensions as lowerCamelCase.
(btw I made mistake in comment above, I typed: upperCamelCase but shown it as loweCamelCase -> of course I meant lowerCamel)